IdentityCore/src/webview/embeddedWebview/MSIDOAuth2EmbeddedWebviewController.m

```diff
@@ -355,6 +356,19 @@ - (void)decidePolicyForNavigationAction:(WKNavigationAction *)navigationAction
         return;
     }
     
+    // Handle anchor links that were clicked
+    if ([navigationAction navigationType] == WKNavigationTypeLinkActivated)
```

Comment:
@ruixi1994 do we override this method in cpp? We might need to verify that it works in msal cpp too.